src/driver/mysql/MysqlDriver.ts

```diff
@@ -310,6 +314,15 @@ export class MysqlDriver implements Driver {
         bigint: { width: 20 },
     }
 
+    /**
+     * Managing version specific data types
+     */
+    columnTypeVersionSupportMap = {
```

Comment:
we don't want to have such feature. If user uses Mariadb version which support all these new types - he can use it. If no - he can't. He needs to put varchar or any other type he thinks is appropriate by himself.

Reply:
Thank you for the feedback.

Sounds appropriate to me. Just so I understand, in the case where a user provides "inet4" but they're on mysql or a version of mariadb that doesn't yet support the data type, let the user receive an error back from the driver?

Reply:
I'm okay with an underlying DBMS throwing the error instead.

Reply:
Sounds good. I did leave the defaulting behavior that was their for mysql for the uuid type since that seems to have some breaking changes. Let me know if this was in error. Thank you again!